cli/src/main.rs

```diff
@@ -246,8 +248,28 @@ fn compile_once(world: &mut SystemWorld, command: &CompileSettings) -> StrResult
     match typst::compile(world) {
         // Export the PDF.
         Ok(document) => {
-            let buffer = typst::export::pdf(&document);
-            fs::write(&command.output, buffer).map_err(|_| "failed to write PDF file")?;
+            if command.output.extension().unwrap_or(OsStr::new("pdf")) == "png" {
```

Comment:
nitpick: this seems like it would be better (as the "pdf" default isn't used)
```suggestion
            if command.output.extension() == Some("png") {
```

Reply:
Didn't know about that, thanks!

Reply:
Ah looks like it needs to be an OsString, will update the PR to reflect that.